packages/search/package.json

```diff
@@ -10,6 +10,7 @@
   "types": "dist/esm/index.d.ts",
   "license": "Apache-2.0",
   "dependencies": {
+    "base-64": "^1.0.0",
```

Comment:
Browsers have built-in fns lik [btoa()](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/btoa) for this, I assume this is needed for node? It's not a big deal to add this I suppose at a whopping [1kb gzipped](https://bundlephobia.com/result?p=base-64@1.0.0). I tried looking into shimming the browser built-ins in a node env, but came across [this comment](https://stackoverflow.com/questions/23097928/node-js-throws-btoa-is-not-defined-error#comment111681927_46524254) which leads me to believe that ultimately it's best to have a consistent implementation on both sides of the wire.

Reply:
Yes your assumption is correct. These are functions used by the Hub Indexer API so it will be in Node env. Also came across the same comment.

Reply:
It could implemented with an interface in which you inject the function to do the base64 encoding, but doesn't now have build in base64 support any ways:
```
Buffer.from("gmike is amazing").toString('base64')
```

Reply:
This is a better approach. I will update to use this.